src/WebJobs.Script/StorageProvider/StorageClientProvider.cs

```diff
@@ -37,7 +37,7 @@ public virtual TClient Create(string name, INameResolver resolver, IConfiguratio
             return this.Create(resolvedName, configuration);
         }
 
-        public virtual TClient Create(string name, IConfiguration configuration)
+        public virtual TClient Create(string name, IConfiguration configuration, TClientOptions options = null)
```

Comment:
Reviewing this code I realize that there's a Sanitization improvement we should make below as well, similar to what we did in https://github.com/Azure/azure-webjobs-sdk/pull/3082 in the StorageClientProvider in the WebJobs SDK.

Reply:
Addressed below.